truss/contexts/image_builder/cache_warmer.py

```diff
@@ -11,10 +11,11 @@
 import boto3
 from botocore import UNSIGNED
 from botocore.client import Config
-from botocore.exceptions import ClientError, NoCredentialsError
 from google.cloud import storage
 from huggingface_hub import hf_hub_download
 
+from truss.util.error_utils import handle_client_error
```

Comment:
FYI: This file is run in a very different context where the truss package is not available. This change broke the integration tests.

Will follow up on monday with the reasoning for this change, but will revert this PR for now.